examples/Cargo.toml

```diff
@@ -11,7 +11,8 @@ cfg-if              = "1.0.0"
 critical-section    = "1.1.2"
 crypto-bigint       = { version = "0.5.5", default-features = false }
 elliptic-curve      = { version = "0.13.8", default-features = false, features = ["sec1"] }
-embassy-executor    = { version = "0.5.0", features = ["task-arena-size-8192"] }
+# some examples use a huge amount of stack
+embassy-executor    = { version = "0.5.0", features = ["task-arena-size-40960"] }
```

Comment:
:exploding_head:

Reply:
I can try something smaller but some of the embassy examples are a bit wasteful with stack currently